modules/rosa-getting-started-access-cluster-web-console.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+[id="rosa-getting-started-access-cluster-web-console_{context}"]
+= Accessing a cluster through the web console
+
+After you have created a cluster admin user or added a user to your configured identity provider, you can log into your {product-title} (ROSA) cluster through the web console.
+
+.Prerequisites
+
+* You have an AWS account.
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a ROSA cluster.
+* You have created a cluster admin user or added your user account to the configured identity provider.
```

Comment:
Thanks Neal for these suggestions.

> Replace "admin" with "administrator."

I have subsequently applied this suggestion where applicable, throughout the PR.

> Consider: since this duplicates the first half of the lead-in sentence, is there value in rewriting that lead-in?

After some reflection, I am going to leave this in the intro and in the prerequisites. I think there is benefit in summarising that you can log into the console after setting up the user requirements.